### PR TITLE
Changed how value is read in flux bootstrap from yaml to allow : char

### PR DESCRIPTION
**Pull request by davidnajar**
Currently, if the yaml contains ':' character in the value field, it is not properly read by the script. Something like this:

```yaml
#cloud-config
#....
flux:
  git:
    url: https://repository.url.com
```

will fail as the git.url param will be read as https (due to using cut ':' in the script). Whit the small change, it will cut at the first ':' and take the rest of the line.

### CODE REVIEW

**Comment by tyzbit** [comment]
Thank you for this PR! I'm not a maintainer so I can't merge it but I originally created the flux bundle and I appreciate the fix. Your change looks good to me.

**Comment by tyzbit** [comment]
I forgot to add - please sign your commit as per the DCO check. The PR can't be merged unless you sign.

**Review by jimmykarily — APPROVED** [review]

**Comment by jimmykarily** [comment]
Looks good, thanks for this PR @davidnajar! Can you please rebase, fix conflicts and fix DCO so we can merge? Thanks.

**Comment by jimmykarily** [comment]
I rebased this and merged it here: https://github.com/kairos-io/community-bundles/commit/c3a401755ce78fef8f074a3cc695cc159d43b224

Part of it was already on `main` but extracting the value to a variable was a nice addition so I kept it. Thanks for this PR @davidnajar!

**Pull request closed without merging**
